### ISSUE TITLE
[IOS] onStateChanged does not work as expected

### ISSUE THREAD

**Issue by leoek** [PSPDFKit/react-native#195]
I wanted to track whether the document was loaded or not to mitigate #194 and according to the source documentation https://github.com/PSPDFKit/react-native/blob/67b572e73cb4df82b57b3045605b64c543e0dc80/index.js#L417 should return an object which includes a property `documentLoaded`.

However it only returns this for me:
```
{"target":1657,"currentPageIndex":0,"annotationEditingActive":false,"pageCount":1,"formEditingActive":false,"annotationCreationActive":false,"textSelectionActive":false}
```

I need to know when the document is loaded to be able to add Annotations to the document.

**Comment by radazzouz** [comment]
Thanks for reporting this, @leoek! `documentLoaded` was indeed missing from the `onStateChanged` payload on iOS. This should be fixed by #197.

**Comment by leoek** [comment]
Thanks for the fast fix!
